client/core/core.go

```diff
@@ -2435,7 +2451,7 @@ func (c *Core) prepareTrackedTrade(dc *dexConnection, form *TradeForm, crypter e
 
 	// Prepare and store the tracker and get the core.Order to return.
 	tracker := newTrackedTrade(dbOrder, preImg, dc, mkt.EpochLen, c.lockTimeTaker, c.lockTimeMaker,
-		c.db, c.latencyQ, wallets, coins, c.notify)
+		c.db, c.latencyQ, wallets, coins, c.notify, c.log)
```

Comment:
We could consider creating a sub-logger for every trade with the order ID in the name, so that it would just be there in all log messages.

Reply:
Meaning that's why passing it instead of using dc's logger?  I think I like the idea of a logger with the order ID to make grepping easier.  Depends on how it turns out.  Can investigate in future work though if you'd like to move forward as-is.